### ISSUE TITLE
[feature request] Find current page in Navigation sidebar

### ISSUE THREAD

**Issue by hariiprasath** [notion-enhancer/notion-enhancer#295]
**Problem**
There is no quick way to find out the hierarchy of where your current page exists. Like if the main pages listed on the sidebar are my "folder" pages and when I am currently viewing some child page, to quickly find out its parents and ancestors is kinda hard right now.

**Solution**
On the sidebar, or near the breadcrumbs in the main panel could be a good spot to put a small button, that, on clicking, expands all the parent folders leading up to the current page.

**Alternatives (optional)**
I have not tried it myself but I have seen the option in Windows Explorer, for the Navigation Panel, to have it "Expand to open folder", so that we are constantly aware of where we are in the sea of pages.

**Comment by dragonwocky** [comment]
what about the breadcrumb at the top?
![image](https://user-images.githubusercontent.com/16874139/100578543-9d957380-3336-11eb-9ebc-36fc16b46f7a.png)

**Comment by horiprasath** [comment]
![image](https://user-images.githubusercontent.com/10820205/100589091-a5deb600-3318-11eb-826b-6ae10f3b06b4.png)
↑ the breacrumbs, although helpful, do not give as much context as an expanded hierarchy this ↓
![image](https://user-images.githubusercontent.com/10820205/100589267-e6d6ca80-3318-11eb-95c3-9629ab76d933.png)
I feel it can be really useful to have an option to have this as well, like in Windows Explorer

**Comment by admiraldus** [comment]
i don't guarantee but i'll try to do this. i think it can be done by simulating the click event.

**Comment by radon-at-beeper** [comment]
Where did this get added? The issue was marked completed, but I wasn't able to find any reference to the requested feature in the current documentation, nor was I able to find any commits that seemed related around the time the issue was closed.

**Comment by dragonwocky** [comment]
@radon-at-beeper it was closed before GitHub let you give a reason to close an issue - I guess when they added the feature, they must have posthumously marked all old closed issues as completed.

I tried a couple of things related to this, and there was some discussion in the Discord server. In the end, though, there was no way to do this at the time without causing layout shift at best and a lot of lag at worst. If the top level hadn't been expanded, there was no good way to tell which one the current page is in without just... trying everything. Now this could probably be done with some API calls to work out which pages need to be expanded. If you'd like to give this a go, I'll reopen the issue and you're free to make a PR.
